docs/sources/datasources/mssql/_index.md

```diff
@@ -136,6 +136,13 @@ You can create queries with the Microsoft SQL Server data source's query editor
 
 For details, refer to the [query editor documentation]({{< relref "./query-editor/" >}}).
 
+### Dataset and Table selection
+
+In the dataset dropdown, choose the MsSQL database to query. The dropdown is be populated with the databases that the user has access to.
+When the dataset is selected, the table dropdown is populated with the tables that are available.
+
+NOTE: If a default database has been chosen through the Data Source configuration page, the Dataset Dropdown will be disabled, and only that configured database will be available for querying.
```

Comment:
Better to bold `Note` as opposed to capitalizing it

Reply:
Agreed. Updated.